src/orca/services/base/ops.py

```diff
@@ -1,27 +1,38 @@
 from abc import ABC, abstractmethod
-from typing import Any, Generic, TypeVar
+from typing import Any, ClassVar, Generic, Type, TypeVar
 
 from pydantic import ConfigDict
 from pydantic.dataclasses import dataclass
 from typing_extensions import Self
 
-from orca.services.base.config import BaseServiceConfig
+from orca.services.base.config import BaseConfig
```

Comment:
I agree with this change, but can you speak to why you renamed `BaseServiceConfig` to `BaseConfig`?

Reply:
Mostly for consistency. I didn't like that the base class was `BaseServiceConfig` whereas the implementations would be `SevenBridgesConfig` and `NextflowTowerConfig` (_i.e. without `Service`, because it's redundant and too long). I prefer `Base` being the only part that is changed when inheriting from a base class.